src/solvers/jesse_dg/ESDG_2D.jl

```diff
@@ -0,0 +1,225 @@
+using OrdinaryDiffEq
+using Trixi
+using Plots
+
+using UnPack
+using StructArrays
+using StructArrays: components
+using StaticArrays
+using RecursiveArrayTools
+using ArrayInterface
+using LazyArrays
+using LinearAlgebra
+using BenchmarkTools
+
+using Octavian
+using LoopVectorization
+using CheapThreads
+
+using StartUpDG
+
+include("/Users/jessechan/.julia/dev/Trixi/src/solvers/jesse_dg/ESDG_utils.jl") # some setup utils
```

Comment:
```suggestion
include("ESDG_utils.jl") # some setup utils
```
To allow others to run the code